backends-clickhouse/src/main/scala/io/glutenproject/utils/ExpressionDocUtil.scala

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.utils
+
+import io.glutenproject.expression.ExpressionMappings
+
+import org.apache.spark.sql.catalyst.analysis.FunctionRegistry
+import org.apache.spark.sql.catalyst.expressions.ExpressionInfo
+
+import java.io.{File, PrintWriter}
+import java.util
+
+object ExpressionDocUtil {
+  val markdownFormat: String = "| %s | %s | %s | %s | %s |\n"
+  val alreadySupported: Map[String, String] = {
+    ExpressionMappings.expressionsMap.map(x => (x._1.getName, x._2))
+  }
+
+  val groupedExpressions: util.Map[String, util.List[ExpressionInfo]] = {
+    val groups = new util.HashMap[String, util.List[ExpressionInfo]]()
+    FunctionRegistry.expressions.foreach(
+      expression => {
+        val (expr, _) = expression._2
+        groups.computeIfAbsent(expr.getGroup, _ => new util.ArrayList[ExpressionInfo]()).add(expr)
+      })
+    groups
+  }
+
+  def main(args: Array[String]): Unit = {
+
+    val group_ordered = new util.ArrayList[String](groupedExpressions.keySet())
+    util.Collections.sort(group_ordered)
+
+    val content = new StringBuffer()
+    val summary = new StringBuffer()
+    content.append(
+      markdownFormat.format("Group", "Expression", "Spark Class Name", "CH Backend", "Examples"))
+    content.append(markdownFormat.format("----", "----", "----", "----", "----"))
+
+    var total = 0
+    var support = 0
+    group_ordered.forEach(
+      name => {
+        var g_total_cnt = 0
```

Comment:
Please use camel-case stype for variables. Same for below.

Reply:
fix